tools/ci_connector_ops/ci_connector_ops/qa_checks.py

```diff
@@ -21,36 +23,32 @@ def check_documentation_file_exists(connector: Connector) -> bool:
 
     return connector.documentation_file_path.exists()
 
+
 def check_documentation_follows_guidelines(connector: Connector) -> bool:
     """Documentation guidelines are defined here https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw"""
     follows_guidelines = True
     with open(connector.documentation_file_path) as f:
-        doc_lines = [l.lower() for l in f.read().splitlines()]
+        doc_lines = [line.lower() for line in f.read().splitlines()]
     if not doc_lines[0].startswith("# "):
         print("The connector name is not used as the main header in the documentation.")
         follows_guidelines = False
-    if connector.definition: # We usually don't have a definition if the connector is not published.
+    if connector.definition:  # We usually don't have a definition if the connector is not published.
```

Comment:
super nitpik (not blocking): while this pr adds an additional space which is required for pep 8 inline comments, not using inline comments is preferred